.gitignore

```diff
@@ -9,6 +9,8 @@ xcuserdata
 /platform/node/test/actual
 /platform/node/test/diff
 /mason_packages
+/benchmark/fixtures/api/cache.db
+/benchmark/fixtures/api/cache.db-*
```

Comment:
Gonna remove this. But we need to find a way of ignoring changes on this database, otherwise it will ways show as "changed" every time we run the tests. Shall we work on a copy? /cc @ivovandongen